docker-compose/docker-compose.yml

```diff
@@ -7,6 +7,19 @@ services:
       - 15672:15672
       - 5672:5672
 
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.8.6
+    ports:
+      - 9200:9200
+      - 9300:9300
+    environment:
+      -  discovery.type=single-node
+
+  kibana:
+    image: docker.elastic.co/kibana/kibana:6.8.6
+    ports:
+      - 5601:5601
+
```

Comment:
Shouldn't we add :
    links:
      - elasticsearch
    depends_on:
      - elasticsearch